src/api/admin.ts

```diff
@@ -289,6 +295,20 @@ const createFirestoreVenueInput_v2 = async (
   return firestoreVenueInput;
 };
 
+const createFirestoreVenueInput_v2 = async (
+  input: VenueInput_v2,
+  user: firebase.UserInfo
+) => {
+  // We temporarily cast the result to unknown so that we can cast to the
+  // same type with the ID added, then we add the missing property.
+  const result = ((await createFirestoreVenueInputWithoutId_v2(
+    input,
+    user
+  )) as unknown) as FirestoreVenueInput_v2;
```

Comment:
I don't like this. I couldn't think of a cleaner way to do this though.
The return from `createFirestoreVenueInputWithoutId_v2` does not have an ID. We want to add that ID property on to both the object and the type of that object. Open to ideas on cleaner ways to do this.